style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+@media (max-width: 375px) {
```

Comment:
Please do not use media queries on mobile view as it's a mobile-first design approach.